src/sync-n-parent-clone.js

```diff
@@ -0,0 +1,20 @@
+var instantiatorCounter = 0;
+AFRAME.registerComponent('instantiator',
+{
+	schema: {
+		on: {type: 'string'},
+		mixin: {type: 'string'}
```

Comment:
Sorry, I should have cleaned things up a bit more to make it clear. This component no longer concerns itself with sync at all. It only turns the entity into a "spawner" of sorts. So these are local mixins